recipe/meta.yaml

```
@@ -47,7 +46,7 @@ requirements:
     - typing-extensions >=4.4.0
   run:
     - python
-    - {{ pin_compatible('numpy') }}
+    - numpy >=2.0,<2.3
```

```suggestion
    - numpy >=1.20
```
https://github.com/PyTables/PyTables/blob/v3.10.2/pyproject.toml#L74
We can follow upstream. The python set in host also has run_exports enforcing an appropriate lower bound.